portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -19,6 +24,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Print the value so you can see it in the server logs.
     System.out.println(textValueName + " submitted contact email: " + textValueEmail);
```

Comment:
Now that you are persisting the data, it would be good to check is the email is in a email format. Another thing to remember when storing user provided content is to make sure that it is sanitized. You don't want to store some malicious script and run that when you fetch the data later. Here is some info on that from the person who originally wrote the tutorial for SPS - https://happycoding.io/tutorials/java-server/sanitizing-user-input